src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
Yep, I meant `GeometricData::None`. I think my preference is to map OSL color parameters to `Color3fPlug( interpretation = None )`, so that we can remove the special case from the serialisation code and still get a concise serialisation. We've already encoded the "colorness" in the type of the plug and the type of the data coming from it, so I don't see the need to double up. I might even argue that we shouldn't have made `Color` an interpretation at all, since it's not really geometric...but that can wait for another Cortex version...